tests/package/test_packagers_manager.py

```diff
@@ -280,3 +293,71 @@ def test_clear_packagers_outputs():
     # Remove remained directory (we tested the clearance of a file and a directory, so we need to delete the directory
     # of the cleared file (it's directory was not marked as future clear)):
     shutil.rmtree(a_temp_dir)
+
+
+@pytest.mark.parametrize(
+    "test_tuple",
+    [
+        (
+            "*list_",
+            [0.12111, 0.56111],
+            {"list_0": 0.12, "list_1": 0.56},
+        ),
+        (
+            "*set_",
+            {0.12111, 0.56111},
+            {"set_0": 0.12, "set_1": 0.56},
+        ),
+        (
+            "*",
+            (0.12111, 0.56111),
+            {"0": 0.12, "1": 0.56},
+        ),
+        (
+            "*error",
+            0.12111,
+            "The log hint key '*error' has an iterable unpacking prefix ('*')",
+        ),
+        (
+            "**dict_",
+            {"a": 0.12111, "b": 0.56111},
+            {"dict_a": 0.12, "dict_b": 0.56},
+        ),
+        ("**", {"a": 0.12111, "b": 0.56111}, {"a": 0.12, "b": 0.56}),
+        (
+            "**error",
+            0.12111,
+            "The log hint key '**error' has a dictionary unpacking prefix ('**')",
+        ),
+    ],
+)
+def test_arbitrary_log_hint(
+    test_tuple: Tuple[
+        str, Union[List[float], Dict[str, float]], Union[Dict[str, float], str]
+    ]
+):
+    """
+    Test the arbitrary log hint key prefixes "*" and "**".
+
+    :param test_tuple: A tuple of the key to use in the log hint, the object to pack and the expected results that
+                       should be packed. A string means an error should be raised.
+    """
+    # Prepare the test:
+    key, obj, expected_results = test_tuple
```

Comment:
```suggestion

@pytest.mark.parametrize(
    "key,obj,expected_results,expected_error",
    [
        (
            "*list_",
            [0.12111, 0.56111],
            {"list_0": 0.12, "list_1": 0.56},
        ),
        (
            "*set_",
            {0.12111, 0.56111},
            {"set_0": 0.12, "set_1": 0.56},
        ),
        (
            "*",
            (0.12111, 0.56111),
            {"0": 0.12, "1": 0.56},
        ),
        (
            "*error",
            0.12111,
            "The log hint key '*error' has an iterable unpacking prefix ('*')",
        ),
        (
            "**dict_",
            {"a": 0.12111, "b": 0.56111},
            {"dict_a": 0.12, "dict_b": 0.56},
        ),
        ("**", {"a": 0.12111, "b": 0.56111}, {"a": 0.12, "b": 0.56}),
        (
            "**error",
            0.12111,
            "The log hint key '**error' has a dictionary unpacking prefix ('**')",
        ),
    ],
)
def test_arbitrary_log_hint(
    key: str, object: ... , expected_results: Dict[str, float] , expected_error: str = None
):
    """
    Test the arbitrary log hint key prefixes "*" and "**".

    :param key:  key to use in the log hint
    :param object: the object to pack
    :param expected_results:  expected results that should be packed. empty if error is expected
    :param expected_error: error message if expected error, empty otherwise
    """
```